src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -10,8 +10,9 @@
 Copyright Contributors to the Zowe Project.
 """
 
-from zowe.core_for_zowe_sdk import SdkApi
+from zowe.core_for_zowe_sdk import SdkApi, constants
```

Comment:
Please update the import to reference `constants` from the same SDK, not from the Core SDK.